### PR TITLE
Use gotty-client instead of termjs-cli

### PR DESCRIPTION
**Pull request by QuentinPerez**
#160

:warning: Don't merge yet :warning:

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -179,38 +180,15 @@ func RemoveDuplicates(elements []string) []string {
 	return result
 }
 
-const termjsBin string = "termjs-cli"
-
-// AttachToSerial tries to connect to server serial using 'term.js-cli' and fallback with a help message
+// AttachToSerial tries to connect to server serial using 'gotty-client' and fallback with a help message
 func AttachToSerial(serverID string, apiToken string, attachStdin bool) error {
-	termjsURL := fmt.Sprintf("https://tty.cloud.online.net?server_id=%s&type=serial&auth_token=%s", serverID, apiToken)
+	URL := fmt.Sprintf("https://tty.cloud.online.net/?arg=%s&arg=%s", apiToken, serverID)
```
Comment (moul) [inline pkg/utils/utils.go]: The URL will be different

Comment (QuentinPerez) [inline pkg/utils/utils.go]: Yes :smile: I will update it

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -179,38 +180,15 @@ func RemoveDuplicates(elements []string) []string {
...
-	}
-	args = append(args, termjsURL)
-	log.Debugf("Executing: %s %v", termjsBin, args)
-	// FIXME: check if termjs-cli is installed
-	spawn := exec.Command(termjsBin, args...)
-	spawn.Stdout = os.Stdout
-	spawn.Stdin = os.Stdin
-	spawn.Stderr = os.Stderr
-	err := spawn.Run()
+	gottycli, err := gottyclient.NewClient(URL)
```
Comment (moul) [inline pkg/utils/utils.go]: We will take advantage of this migration to pass context streams:

- [ ] blocked by https://github.com/moul/gotty-client/issues/21
- [ ] update AttachToSerial to take `ctx` or streams as argument

File: pkg/utils/utils.go [hunk 3]
```diff
@@ -179,38 +180,15 @@ func RemoveDuplicates(elements []string) []string {
...
-You need to install '%s' from https://github.com/moul/term.js-cli
-
-    npm install -g term.js-cli
-
-However, you can access your serial using a web browser:
-
-    %s
-
-`, termjsBin, termjsURL)
 		return err
```
Comment (moul) [inline pkg/utils/utils.go]: We need to write the "Press ctrl+Q to quit".

File: pkg/utils/utils.go (hunk 2 above)
Comment (moul) [inline pkg/utils/utils.go]: Also ensure that `scw -D` will print out the URL

**Comment by QuentinPerez** [comment]
Done

**Comment by moul** [comment]
Can you update the README.md ?

**Comment by moul** [comment]
LGTM

**Pull request merged**